src/bin/render-readmes.rs

```diff
@@ -137,7 +137,7 @@ fn main() {
                         content,
                         content_length,
                         "text/html",
-                        None,
+                        header::HeaderMap::new(),
```

Comment:
Shouldn't we set a TTL here as well?

(For context, this is a binary that re-renders all readme files. It can be manually run if we make updates to the readme rendering code.)